README.md

```diff
@@ -1,52 +1,97 @@
-These are example JSONs we expect to receive
+# API suggested specs
 
-##Fairphone
+These are JSON-like specs for the data we expect to receive.
 
-```JSON
+JSON properties are written in the following format: `"key" [quantifier]: valueType`.
+
+- `"key"`: literal key name, lowercaseCamelCase
+- `quantifier`:
+  - [absence]: property is **obligatory**
+  - `*`: 0 or more array items; property is optional
+  - `+`: 1 or more array items; property is **obligatory**
+  - `?`: 0 or 1; property is optional
+  - `{n}`: exactly _n_ ocurrences; property is **obligatory**
+- `valueType`:
+  - `UppercaseCamelCase`: data type
+  - `lowercase_snake_case`: String format
+
+
+## Fairphone
+
+### Angels endpoint
+
+```JavaScript
 {
-  "meetups" :
-  [
+  "angels" *: [
     {
-      "eventName" : "The example party",
-      "startDate" : "2016-10-13T14:00+02:00",
-      "endDate" : "2016-10-13T18:00+02:00",
-      "lat_lng" : [48.210033,16.363449],
-      "website" : "https://fairphone.com/meetups/The-example-party/12"
+      "cityName": String,
+      "countryName" ?: String,
+      "coordinates": {
+        "latitude": Float,
+        "longitude": Float
+      },
+      "email": email,
+      "people" *: [
+        {
+          "name" ?: String,
+          "forumNick" ?: String
+        },
+        {...}
+      ]
     },
-    {}  
-  ],
-  "angels" :
-  [
+    {...}
+  ]
+}
+```
+
+
+### Meetups endpoint
+
+```JavaScript
+{
+  "meetups" *: [
     {
-      "cityName" : "Foo City",
-      "lat_lng" : [48.210033,16.363449],
-      "website" : "https://fairphone.com/angels/foo-city/36"
+      "eventName": String,
+      "startDate": unix_time,
```

Comment:
I just looked up what `unix_time` actually is, and it is not intuitive at all! So I don't see any reason for choosing an abstract row of digits above a human-readable date format like ISO-8601.